opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -24,14 +24,24 @@ option java_package = "io.opentelemetry.proto.metrics.v1";
 option java_outer_classname = "MetricsProto";
 option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/metrics/v1";
 
-// A collection of metrics from a Resource.
+// A collection of InstrumentationLibraryMetrics from a Resource.
 message ResourceMetrics {
-  // A list of metrics that originate from a resource.
-  repeated Metric metrics = 1;
-
   // The resource for the metrics in this message.
   // If this field is not set then no resource info is known.
-  opentelemetry.proto.resource.v1.Resource resource = 2;
+  opentelemetry.proto.resource.v1.Resource resource = 1;
+
+  // A list of metrics that originate from a resource.
+  repeated InstrumentationLibraryMetrics library_metrics = 2;
+}
+
+// A collection of Metrics from a InstrumentationLibrary.
```

Comment:
```suggestion
// A collection of Metrics produced by an InstrumentationLibrary.
```
Otherwise it sounds like this was about "meta-"metrics regarding the instrumentation itself, not the instrumented application.